tuf-spec.md

```diff
@@ -1425,14 +1429,20 @@ it in the next step.
   in the trusted timestamp metadata.  If the versions do not match, discard the
   new snapshot metadata, abort the update cycle, and report the failure.
 
-5. **Check for a rollback attack**. The version number of the targets
-  metadata file, and all delegated targets metadata files, if any, in the
-  trusted snapshot metadata file, if any, MUST be less than or equal to its
-  version number in the new snapshot metadata file. Furthermore, any targets
-  metadata filename that was listed in the trusted snapshot metadata file, if
-  any, MUST continue to be listed in the new snapshot metadata file.  If any of
-  these conditions are not met, discard the new snapshot metadata file, abort
-  the update cycle, and report the failure.
+5. **Check for a rollback attack**.
+
+  1. The new snapshot metadata file MUST contain the description of the targets
```

Comment:
```suggestion
  1. The new snapshot metadata file MUST contain the version numbers of the targets
```
The listed item is the version numbers, not a description.

Reply:
I assume that "description" was used as a short-hand for the version plus optional length and hashes (fields in a `METAFILES` object)? As this terminology is not used elsewhere we should certainly be more exploit here.

Reply:
Perhaps we could just say
>The new snapshot metadata file's `METAFILES` object MUST list all targets metadata (top-level targets.EXT all all delegated roles) including their version numbers, optionally their length and optionally their hashes. If not, discard the new snapshot metadata file, abort the 

etc.